packages/onboarding-ui/src/pages/ConfirmationPage/ConfirmationPage.spec.tsx

```diff
@@ -0,0 +1,15 @@
+import ReactDOM from 'react-dom';
```

Comment:
I think the name of this page component its too generic. We should use something related to the password e.g `PasswordResetConfirmationPage`, `PasswordConfirmationPage`, `PasswordResetPage`, `PasswordResetConfirmationPage`

Reply:
This is to be generic because is a confirmation page that receives a title and subtitle to show a confirmation page that can be used in any place.

Reply:
Dude, we have a bunch of pages that could be versions of a generic one, e.g `MagicLinkEmailPage`, `InvalidLinkPage`, `EmailConfirmedPage`, `ConfirmationProcessPage`, `CheckYourEmailPage` etc. But we decide to follow a pattern to create separated pages using the sharable style. To continue with this generic page approach you're looking for, IMHO we have to remove all these pages and replace them with just this generic one passing the proper props to it

Reply:
I will make this adaptable to the actual pattern and maybe after we can talk about this refactory if this make sense.